openapi/godaddy.abuse/client.bal

```diff
@@ -39,6 +39,7 @@ public isolated client class Client {
         http:Client httpEp = check new (serviceUrl, clientConfig);
```

Comment:
Invalid default service URL has used in the method signature. Please fix if possible.

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/e2ec5928799e05c94828171641d94ac96370565e